framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -108,12 +109,20 @@ XYDelaunayGenerator::validParams()
       "tri_element_type", tri_elem_type, "Type of the triangular elements to be generated.");
   params.addParam<bool>(
       "verbose_stitching", false, "Whether mesh stitching should have verbose output.");
-
+  params.addParam<std::vector<Point>>("interior_points",
+                                      {},
```

Comment:
Do not set default value as `{}` here, use `isParamValid()` to check if the parameter is specified in the constructor.

Reply:
Why is this incorrect? Here the default is an empty vector for interior points. So none will be added.

Reply:
It is correct. But using the other approach could make it easier when you check whether both position parameters are specified.

Reply:
this approach makes it easier to retrieve the vector with getParam

later on for checking whether they are specified, you can use vector.empty() with this approach

the nice advantage of using {} over no default, is that you naturally handle the "param is valid but set by the user as an empty vector"